src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() {
```

Comment:
Command#isMutating() does not have a header comment.
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.

Reply:
So something like this?
/**
     * Returns a boolean to indicate whether the command mutates the stored addressBook data
     *
     * @return boolean from overriding method in subclass, false if subclass does not have an overriding method
     */
    public boolean isMutating() {
        return false;
    }